### DISCUSSION TITLE
More details about code challenge needed.

### DISCUSSION THREAD

**Discussion by gaurab-khanal** [camicroscope/GSOC#15] [Q&A] [answered]
Hi I wanna contribute to 'Modularization of Components' project for GSOC 2023. I am confused more about code challenge. What kind of project should I submit for code challenge? 
For 'Modularization of Components' project the code challenge is  'A simple webapp demonstrating Web Components'. What should I build here? Can I get more specific details what kind of project I should make?
Also, Does code challenge determine for my selection?

**Comment by birm** [comment, marked as answer]
The code challenge is a substantial part of selection.
For code challenges, we're hoping to determine your problem solving ability and style first and foremost. For this project, for example, while a "hello world" web component application would technically meet the requirements, you might want to show that you understand how to handle data flow too.

Hope this helps!

**Reply by ShivanshAr97** [reply]
Hey @birm I have a doubt regarding this:
Like for the code challenge, we have to make the changes in this repository only and share the PR requests or make a new repository in our profile and add various web components like a random landing page or a login page?

**Reply by birm** [reply]
Code challenges should be independent from the project source. They're basically interview questions

**Reply by ShivanshAr97** [reply]
Got it Thanks!
